scripts/start/commands/samples.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   builder: {},
   handler: () => {
     console.info('Starting the samples ...');
-    const command = `webpack-dev-server --config scripts/webpack/webpack.dev.babel.js --hot --inline --history-api-fallback --context ${path.resolve('./samples')}`;
+    const command = `webpack-dev-server --config scripts/webpack/webpack.dev.babel.js --hot --history-api-fallback --context ${path.resolve('./samples')}`;
```

Comment:
What does this `--inline` change in the final command? Could that be the reason for the warnings we are observing?

Reply:
No that's not the reason for warnings, it was due to css-loader version upgrade. This change is required as per webpack 5 config supported